tests/docs/expected.js

```diff
@@ -484,7 +484,7 @@ module.exports = {
     'sort',
     'sortBy',
     'startRouting',
-    'store',
+    '_store',
```

Comment:
ELI5 this file?

Reply:
Every name Ember exports gets checked here as a suuuuper basic "smoke test" to make sure we don't accidentally stop publishing something.

Reply:
Also, there is apparently some *other* usage of `store` so I have reverted this change. 😂